core/templates/core/base.html

```diff
@@ -7,7 +7,7 @@
         <title>GwardiaHub | {% block title %}{% endblock %}</title>
         <script src="https://cdn.tailwindcss.com"></script>
         {% load static %}
-        <script src="{% static "tailwind_custom_styles.js" %}"></script>
+        <script src="{% static 'GwardiaHub/tailwind_custom_styles.js' %}"></script>
```

Comment:
Why? And I think we should use double quotes here...

Reply:
![image](https://github.com/Gwardia-Czapli/GwardiaHub/assets/88268063/d22110bc-4a7c-4544-ac9d-2ae2164751d5)